### PR TITLE
Fix for missing additional features fields when creating new popup windows

### PR DESCRIPTION
**Pull request by MichaelVasseur**
The non-standard features (like this-is-not-a-standard-feature) given to window.open() are lost in the popup window creation workflow. This leads to the 'new-window' event handler not receiving these 'additional' features. These features may be used by applications to give some specific data to the popup.

NOTE : This is a prerequisite of pull request [#7291](https://github.com/electron/electron/pull/7291) in electron repo.

### CODE REVIEW

File: patches/ThomsonReuters_missing_additional_features_fix.patch [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+--- content\public\browser\content_browser_client.h	2016-09-16 09:47:51.000000000 +0200
++++ content\public\browser\content_browser_client.h	2016-09-16 09:33:32.000000000 +0200
+@@ -512,12 +512,13 @@
+                                const GURL& source_origin,
+                                WindowContainerType container_type,
+                                const GURL& target_url,
+                                const Referrer& referrer,
+                                WindowOpenDisposition disposition,
+                                const blink::WebWindowFeatures& features,
++							   const std::vector<base::string16>& additional_features,
```
Comment (zcbenz) [inline patches/ThomsonReuters_missing_additional_features_fix.patch]: Can you replace tabs with spaces?

**Comment by MichaelVasseur** [comment]
Requested changes done.

**Comment by MichaelVasseur** [comment]
@zcbenz PR updated with requested changes

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**